otx/algorithms/common/tasks/training_base.py

```diff
@@ -126,6 +125,27 @@ def __init__(self, task_config, task_environment: TaskEnvironment, output_path:
         # to override configuration at runtime
         self.override_configs = {}  # type: Dict[str, str]
 
+        if self._is_multi_gpu_training():
+            self._setup_multigpu_training()
+
+    @staticmethod
```

Comment:
Why did you set this function as static? It seems that this function is only used with `self.` and this function is a private function.

Reply:
Thanks for answer :)  As Songki said, I also think that static method don't have to be public method. And also being staticmethod doesn't have any side effect but have advantage even by not to need information of self.